pkg/apis/v1/labels.go

```diff
@@ -119,6 +119,7 @@ var (
 	LabelInstanceAcceleratorManufacturer      = apis.Group + "/instance-accelerator-manufacturer"
 	LabelInstanceAcceleratorCount             = apis.Group + "/instance-accelerator-count"
 	AnnotationEC2NodeClassHash                = apis.Group + "/ec2nodeclass-hash"
+	AnnotationKubeletCompatibilityHash        = coreapis.CompatabilityGroup + "/kubelet-drift-hash"
```

Comment:
I don't think we want to use the `coreapis.CompatabilityGroup`, we want to make sure we are staying in the aws group. We should just define the group here https://github.com/aws/karpenter-provider-aws/blob/51122a57614038eda83ada24abad0bbb387259e9/pkg/apis/apis.go#L29